src/common/ui/code.vue

```diff
@@ -69,7 +68,10 @@ import 'codemirror/keymap/sublime';
 import 'codemirror/addon/hint/show-hint.css';
 import 'codemirror/addon/hint/show-hint';
 import 'codemirror/addon/hint/javascript-hint';
+import 'codemirror/addon/hint/html-hint';
+import 'codemirror/addon/hint/css-hint';
```

Comment:
We won't use html/css addons as they add ~10kB, too big for such a rarely used feature. People either use devtools to write CSS/HTML or the specialized editors.

Reply:
It's just a nice to have but it's no big loss.

I'm curious on the reasons that the extra ~10k. Given all the files are served locally rather than network, are you concerned about the extra load / parse time or memory?

Reply:
Violentmonkey's built-in editor is just a stop-gap measure to perform quick edits and it's meant to be as bare-bones as possible so only the more frequently used features are added.

Reply:
okay. I've backed out the commit.